app/services/elasticsearch.js

```diff
@@ -0,0 +1,21 @@
+const elasticsearch = require('elasticsearch');
+
+const config = require('../config').get('elasticsearch');
```

Comment:
1) why parameter name is config? As for me, config is smth global, but here we use only elastic configuration
2) why not require('nconf').get...? We already loaded all configuration in nconf - lets use it in everywhere

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/10/commits/ad6fe36744dfeb0ba729984f04398e75bbd8b8a1#diff-985fae5d0819f6ee40d7b8c6ec4475cfR2). Now receive config from `nconf` and rename to `elasticConfig`